src/py21cmfast/src/SpinTemperatureBox.c

```diff
@@ -1167,12 +1460,28 @@ LOG_SUPER_DEBUG("looping over box...");
 
                         dxlya_dt_box[box_ct] *= const_zp_prefactor*prefactor_1 * (1.+delNL0[0][box_ct]*growth_factor_zp);
                         dstarlya_dt_box[box_ct] *= prefactor_2;
+                        if (flag_options->USE_MINI_HALOS){
+                            dstarlyLW_dt_box[box_ct] *= prefactor_2 * (hplank * 1e21);
```

Comment:
we will be using J21 to calculate Mcrit for photodissociation. This value is in order of 1e-21, therefore, I multiply 1e21 here in case float variables cannot record this small value.

Reply:
Ah I see, so it's not units, just normalization. Thanks!